_includes/setup/android-setup-linux.md

```diff
@@ -0,0 +1,89 @@
+## Android setup
```

Comment:
Thanks for asking! Considered the nested include file approach. Not easy to maintain, though. For new contributors and team members, it's best to keep the organization simple. That's one reason. Another reason is the main drawback in this organization would be if we expected the Android setup instructions to change frequently. Then, each content change would need to be double-committed. In comparison to the other setup content, this piece seems more stable. Thus, not much down side to this approach.